plugins/destination/bigquery/client/client_test.go

```diff
@@ -8,14 +8,14 @@ import (
 )
 
 func TestPlugin(t *testing.T) {
-	p := destination.NewPlugin("bigquery", "development", New)
+	p := destination.NewPlugin("bigquery", "development", New, destination.WithManagerWriter())
```

Comment:
I realize now there's a typo in this name, it's `WithManagerWriter` but probably should be `WithManagedWriter` 😮‍💨

Reply:
https://github.com/cloudquery/plugin-sdk/pull/526